examples/mouse_eye_movements.py

```diff
@@ -0,0 +1,300 @@
+"""Pupil tracking
+=================
+
+Look at eye movements caused by the oculo-motor reflex.
```

Comment:
```suggestion
Look at eye movements and pupil diameter.
```

I suggest mentioning the reflex later, e.g. when looking at the velocity plot.